### DISCUSSION TITLE
Completely uninstall Prism Launcher with all files it created and changed in macOS

### DISCUSSION THREAD

**Discussion by EchoEllet** [PrismLauncher/PrismLauncher#2093] [Q&A] [answered]
Completely uninstall Prism Launcher with all files it created alongside with Minecraft it installed, like I was never installing the launcher, I don't want to uninstall the launcher but I'm just asking if there is script to do such thing in macOS

I installed it using home-brew

**Comment by Scrumplex** [comment, marked as answer]
The only folder Prism Launcher creates files in is at `~/Library/Application Support/PrismLauncher`. It includes everything from jar files, instances, settings and accounts. So deleting that should be all you need

**Reply by EchoEllet** [reply]
![image](https://github.com/PrismLauncher/PrismLauncher/assets/73608287/a78bcf69-15f3-4ee6-92f1-8e8a25f08580)
It looks like there was more, I don't know if there are even more, it would be very helpful to make the uninstall built-in to help track of all the files that Prism launcher create

**Comment by Aces-and-Jacks** [comment]
`/Applications/Prism Launcher.app`

`/Users/****/Library/Application Support/CrashReporter/prismlauncher_C63797B8-6010-5262-BD96-145BEDDF1A07.plist`

`/Users/****/Library/Application Support/PrismLauncher`

`/Users/****/Library/Caches/org.prismlauncher.PrismLauncher`

`/Users/****/Library/HTTPStorages/org.prismlauncher.PrismLauncher`

`/Users/****/Library/Logs/DiagnosticReports/Retired/prismlauncher-2024-01-30-194611.ips`

`/Users/****/Library/Preferences/org.prismlauncher.PrismLauncher.plist`

`/Users/****/Library/Saved Application State/org.prismlauncher.PrismLauncher.savedState`

`/Users/****/Library/WebKit/org.prismlauncher.PrismLauncher`

`/private/var/folders/4k/t41mm2cj7n99_bbf6qgtnb3h0000gr/T/org.prismlauncher.PrismLauncher`

`/private/var/folders/4k/t41mm2cj7n99_bbf6qgtnb3h0000gr/C/org.prismlauncher.PrismLauncher`


@ellet0 These are all the folders I found made by prism launcher, if you want a complete uninstall. Two helpful apps helped me find them. One is [AppCleaner](https://freemacsoft.net/appcleaner/) and the other is [App Cleaner & Uninstaller](https://nektony.com/mac-app-cleaner). The first one is free and what I mainly use to delete apps (rather than just deleting the app icon and application support folder), while the second one is a subscription and I only use the free version to find file paths not detected by the first, which shows a popup after I move the files I want to delete to trash.

**Reply by EchoEllet** [reply]
> `/Applications/Prism Launcher.app`
> 
> `/Users/****/Library/Application Support/CrashReporter/prismlauncher_C63797B8-6010-5262-BD96-145BEDDF1A07.plist`
> 
> `/Users/****/Library/Application Support/PrismLauncher`
> 
> `/Users/****/Library/Caches/org.prismlauncher.PrismLauncher`
> 
> `/Users/****/Library/HTTPStorages/org.prismlauncher.PrismLauncher`
> 
> `/Users/****/Library/Logs/DiagnosticReports/Retired/prismlauncher-2024-01-30-194611.ips`
> 
> `/Users/****/Library/Preferences/org.prismlauncher.PrismLauncher.plist`
> 
> `/Users/****/Library/Saved Application State/org.prismlauncher.PrismLauncher.savedState`
> 
> `/Users/****/Library/WebKit/org.prismlauncher.PrismLauncher`
> 
> `/private/var/folders/4k/t41mm2cj7n99_bbf6qgtnb3h0000gr/T/org.prismlauncher.PrismLauncher`
> 
> `/private/var/folders/4k/t41mm2cj7n99_bbf6qgtnb3h0000gr/C/org.prismlauncher.PrismLauncher`
> 
> 
> @ellet0 These are all the folders I found made by prism launcher, if you want a complete uninstall. Two helpful apps helped me find them. One is [AppCleaner](https://freemacsoft.net/appcleaner/) and the other is [App Cleaner & Uninstaller](https://nektony.com/mac-app-cleaner). The first one is free and what I mainly use to delete apps (rather than just deleting the app icon and application support folder), while the second one is a subscription and I only use the free version to find file paths not detected by the first, which shows a popup after I move the files I want to delete to trash. 

Yes I found those too, this is why I asked
